### ISSUE TITLE
PolyplopiaSegment issue at Detector Level (Trac #1851)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1843]
Hi,

I'm trying to run PolyplopiaSegment on the simulation release V04-01-13 at detector level. I end up with this error:

`/data/user/saxani/environments/buildDet//env-shell.sh: line 155: 26646 Bus error               PATH=$_PATH LD_LIBRARY_PATH=$_LD_LIBRARY_PATH DYLD_LIBRARY_PATH=$_DYLD_LIBRARY_PATH PYTHONPATH=$_PYTHONPATH ICETRAY_CLASSPATH=$_ICETRAY_CLASSPATH I3_PLATFORM="" I3_SRC=$_I3_SRC I3_BUILD=$_I3_BUILD I3_PORTS=$_I3_PORTS I3_TESTDATA=$_I3_TESTDATA ROOTSYS=/cvmfs/icecube.opensciencegrid.org/py2-v1/RHEL_6.0_amd64/i3ports/root-v5.30.06 I3_SHELL=$_I3_SHELL $NEW_SHELL $ARGV`

Can we reimplement the older version of PolyplopiaSegment into this release so that it can run after photon propagation?

ATTN: Juan Carlos

Thanks,

Spencer

Migrated from https://code.icecube.wisc.edu/ticket/1851
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:12:54", 
    "description": "Hi,\n\nI'm trying to run PolyplopiaSegment on the simulation release V04-01-13 at detector level. I end up with this error:\n\n`/data/user/saxani/environments/buildDet//env-shell.sh: line 155: 26646 Bus error               PATH=$_PATH LD_LIBRARY_PATH=$_LD_LIBRARY_PATH DYLD_LIBRARY_PATH=$_DYLD_LIBRARY_PATH PYTHONPATH=$_PYTHONPATH ICETRAY_CLASSPATH=$_ICETRAY_CLASSPATH I3_PLATFORM=\"\" I3_SRC=$_I3_SRC I3_BUILD=$_I3_BUILD I3_PORTS=$_I3_PORTS I3_TESTDATA=$_I3_TESTDATA ROOTSYS=/cvmfs/icecube.opensciencegrid.org/py2-v1/RHEL_6.0_amd64/i3ports/root-v5.30.06 I3_SHELL=$_I3_SHELL $NEW_SHELL $ARGV`\n\nCan we reimplement the older version of PolyplopiaSegment into this release so that it can run after photon propagation?\n\nATTN: Juan Carlos\n\nThanks,\n\nSpencer", 
    "reporter": "saxani", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1550067174476394", 
    "component": "combo simulation", 
    "summary": "PolyplopiaSegment issue at Detector Level", 
    "priority": "normal", 
    "keywords": "", 
    "time": "2016-09-06T14:31:56", 
    "milestone": "", 
    "owner": "juancarlos", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20160906T14:47:19: 
* cweaver changed component from "cmake" to "simulation"
* cweaver changed owner from "" to "juancarlos"
* cweaver commented:

> Assign this to juancarlos so he sees it. 


* cweaver changed status from "new" to "assigned"

**Comment by icecube-trac** [comment]
Trac update at 20161103T21:23:57: juancarlos commented:

> Is this still needed?

**Comment by icecube-trac** [comment]
Trac update at 20161220T16:05:25: 
* olivas commented:

> Chatted with Spencer and Juan Carlos separately.  This might not be needed any more.  Spencer will know more in a week or so.  We'll revisit in January.


* olivas changed milestone from "" to "January 2017"

**Comment by icecube-trac** [comment]
Trac update at 20170223T22:53:00: 
* juancarlos changed status from "assigned" to "closed"
* juancarlos commented:

> In changeset 153504:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="153504"
> Added back functionality in polyplopia for merging pregenerated MCPEs. Closes #1843
> ```


* juancarlos changed resolution from "" to "fixed"

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:12:54: 
* olivas commented:

> Milestone January 2017 deleted


* olivas changed milestone from "January 2017" to ""
